examples/tasks_tutorial.py

```diff
@@ -58,7 +58,9 @@
 print(len(filtered_tasks))
 
 ############################################################################
-# Resampling strategies can be found on the `OpenML Website <http://www.openml.org/search?type=measure&q=estimation%20procedure>`_.
+# Resampling strategies can be found on
+# the `OpenML Website <http://www.openml.org
```

Comment:
@mfeurer I checked something there, noticed that the line was too long and I fixed it. The problem now, would be that reversing it, would mean it is on the diff and then it would fail the build because it is >90. Or am I wrong ?

Reply:
It should not appear in the diff because the file will stay the same compared to the development branch.